Observer/SalesOrderShipmentTrackDeleteAfter.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/*
+ * Copyright Extend (c) 2022. All rights reserved.
+ * See Extend-COPYING.txt for license details.
+ */
+
+namespace Extend\Integration\Observer;
+
+use Extend\Integration\Service\Api\Integration;
+use Extend\Integration\Service\Api\ShipmentObserverHandler;
+use Magento\Framework\Event\ObserverInterface;
+use Magento\Framework\Event\Observer;
+use Magento\Store\Model\StoreManagerInterface;
+use Magento\Sales\Model\Order\Shipment;
+use Magento\Sales\Model\Order\Shipment\Track;
+use Psr\Log\LoggerInterface;
+
+class SalesOrderShipmentTrackDeleteAfter implements ObserverInterface
+{
+  /**
+   * @var LoggerInterface
+   */
+  private $logger;
+
+  /**
+   * @var ShipmentObserverHandler
+   */
+  private $shipmentObserverHandler;
+
+  /**
+   * @var Integration
+   */
+  private $integration;
+
+  /**
+   * @var StoreManagerInterface
+   */
+  private $storeManager;
+
+  public function __construct(
+    LoggerInterface $logger,
+    ShipmentObserverHandler $shipmentObserverHandler,
+    Integration $integration,
+    StoreManagerInterface $storeManager
+  ) {
+    $this->logger = $logger;
+    $this->shipmentObserverHandler = $shipmentObserverHandler;
+    $this->integration = $integration;
+    $this->storeManager = $storeManager;
+  }
+
+  /**
+   * @param Observer $observer
+   * @return void
+   */
+  public function execute(Observer $observer)
+  {
+    try {
+      /** @var Track */
+      $track = $observer->getEvent()->getTrack();
+      /** @var Shipment */
+      $shipment = $track->getShipment();
+      $endpoint = ['path' => Integration::EXTEND_INTEGRATION_ENDPOINTS['webhooks_shipments_update'], 'type' => 'middleware'];
```

Comment:
Just confirming, it is supposed to be `webhooks_shipments_update` and not `webhooks_shipments_delete`?

Reply:
Correct, we want the corresponding shipment to be updated to reflect the track being removed. Node core will fetch the shipment from magento to get its current state (which will include the updated track list)